tests/TestCase/ORM/ResultSetTest.php

```diff
@@ -416,4 +417,40 @@ public function testIsEmptyDoesNotConsumeData()
         $res->isEmpty();
         $this->assertCount(6, $res->toArray());
     }
+
+    /**
+     * Test that ResultSet
+     *
+     * @return void
+     */
+    public function testCollectionMinAndMax()
+    {
+        $query = $this->table->find('all');
+
+        $min = $query->min('id');
+        $minExpected = $this->table->get(1);
+
+        $max = $query->max('id');
+        $maxExpected = $this->table->get(3);
+
+        $this->assertEquals($minExpected, $min);
+        $this->assertEquals($maxExpected, $max);
+    }
+
+    /**
+     * Test that ResultSet
+     *
+     * @return void
+     */
+    public function testCollectionMinAndMaxWithAggregateField()
+    {
+        $query = $this->table->find();
+        $query->select(['count' => 'length(title)']);
```

Comment:
This causes failure on SQL Server which doesn't have `length()` function but `len()` instead.

Reply:
My bad. Fixed in 0da10887762b489c34a264afd675bc7e3fca4c56.

Reply:
No 😊 

```
1) Cake\Test\TestCase\ORM\ResultSetTest::testCollectionMinAndMaxWithAggregateField
PDOException: SQLSTATE[42S22]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Invalid column name 'author_id + 2'.
```

Reply:
Darn. I'll try again :smile: